job/job.go

```diff
@@ -86,6 +94,30 @@ type Job struct {
 	IsDone bool `json:"is_done"`
 }
 
+type jobType int
+
+const (
+	LocalJob jobType = 0 + iota
+	RemoteJob
+)
+
```

Comment:
Can we add a comment describing `RemoteProperties`, can be the same as the above one. Mostly for the godocs generated html.